js/client/modules/@arangodb/testsuites/permissions_server.js

```diff
@@ -84,22 +84,35 @@ function permissions_server(options) {
 
         paramsSecondRun = executeScript(content, true, testFile);
       } catch (ex) {
+        let shutdownStatus = pu.shutdownInstance(instanceInfo, clonedOpts, false);                                     // stop
         results[testFile] = {
           status: false,
-          messages: 'Warmup of system failed: ' + ex
+          messages: 'Warmup of system failed: ' + ex,
+          shutdown: shutdownStatus
         };
-        pu.shutdownInstance(instanceInfo, clonedOpts, false);                                     // stop
+        results['shutdown'] &= shutdownStatus;
```

Comment:
Please do not use a bitwise operator for a logical operation just for verbosity.
```suggestion
        results['shutdown'] = results['shutdown'] && shutdownStatus;
```